src/utils/cancelRequests.ts

```diff
@@ -0,0 +1,35 @@
+import axios, { CancelTokenSource, AxiosRequestConfig, CancelToken as CancelTokenAxios } from 'axios';
+import { RequestConfiguration } from '../layer/const';
```

Comment:
Not sure why `npm run build` doesn't warn about it, but there is a circular import here (`src/utils/cancelRequests.ts` => `src/layer/const.ts` => `src/utils/cancelRequests.ts`). I think it doesn't pose any problems in this case, but I would still suggest moving `RequestConfiguration` to this file - it belongs here a bit more anyway (imho).

Nitpick: by convention we use absolute imports in this library (`from 'src/layer/const';` instead of `from '../layer/const'`).